cfme/tests/intelligence/reports/test_metering_report.py

```diff
@@ -261,6 +265,7 @@ def metering_report(vm_ownership, provider):
 # Tests to validate usage reported in the Metering report for various metrics.
 # The usage reported in the report should be approximately equal to the
 # usage estimated in the resource_usage fixture, therefore a small deviation is fine.
+@pytest.mark.uncollectif(lambda provider: provider.category == 'cloud')
```

Comment:
Question : Cloud support is added in this PR , any reason why this test is skipped for cloud provider

Reply:
That metric is not supported for cloud providers. I have added a comment about it in one of the newer commits.

Reply:
:+1: